android-demos/GeotriggerMonitoring/README.md

```diff
@@ -0,0 +1,58 @@
+# android-geotrigger-monitoring-demo
+
+This code sample demonstrates monitoring Geotriggers in a Foreground Service on Android. Learn more in the associated [blog post]().
+
+The directory `GeotriggerMonitoringDemo-WithoutGeotriggers` demonstrates the app before adding Geotrigger monitoring, whereas the directory `GeotriggerMonitoringDemo-WithGeotriggers` demonstrates the app after adding Geotrigger monitoring.
+
+![android-geotrigger-monitoring-demo](android-geotrigger-monitoring-demo.png)
+
+## Features
+* Geotriggers - Monitor device location as it enters and exits areas of interest
+* Android best practices - Uses a clean architecture approach to using the ArcGIS Runtime SDK for Android
+
+## Instructions
+
+1. Fork and then clone the repo.
+2. Set your api key in the `app/res/values/strings.xml` file.
```

Comment:
Might need some better pointers for the customer here.  

Samples use the following statement;
API key: A permanent key that gives your application access to Esri location services. Visit your ArcGIS Developers [Dashboard](https://developers.arcgis.com/dashboard) to create a new API key or access an existing API key.